### PR TITLE
{cae,numlib,phys}[foss/2019b] GATE v9.0, Geant4 v10.6, CLHEP v2.4.1.3

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

requires #10514 (`ROOT`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 3 (3 easyconfigs in this PR)
node2665.swalot.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/6328e1b59ef2e4b23d84b52c67f761d5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 3 (3 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, x86_64, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/3d0f83fe6b8bb9b183fe0d65d0b9d742 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 3 (3 easyconfigs in this PR)
node3128.skitty.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/a2b72ad9fd906a185353c57b00de6d43 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 3 (3 easyconfigs in this PR)
node3402.kirlia.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/d84c4eb6c2942cf918632b2e1dee1f9f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3402.kirlia.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/0cba51697fc38d231238a274f6abe0ab for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3128.skitty.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/a7b7673ea69efa60229d8ff9eb5120ed for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2698.swalot.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/b71a13bfeb8b110d62cd594e644c8857 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3401.kirlia.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/778ed8df766bf81c771b0ee1ecea8035 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node377.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/e747cdaefbfca20b6e971389e8a2d673 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node125.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/07d0fb2f5624b6ac16b0249f38a17637 for a full test report.

File: easybuild/easyconfigs/c/CLHEP/CLHEP-2.4.1.3-foss-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+easyblock = 'CMakeMake'
+
+name = 'CLHEP'
+version = '2.4.1.3'
+
+homepage = 'http://proj-clhep.web.cern.ch/proj-clhep/'
```
Comment (lexming) [inline easybuild/easyconfigs/c/CLHEP/CLHEP-2.4.1.3-foss-2019b.eb:6]:
```suggestion
homepage = 'https://proj-clhep.web.cern.ch/proj-clhep/'
```

File: easybuild/easyconfigs/g/Geant4/Geant4-10.6-foss-2019b.eb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name = 'Geant4'
+version = '10.6'
+
+homepage = 'http://geant4.cern.ch/'
```
Comment (lexming) [inline easybuild/easyconfigs/g/Geant4/Geant4-10.6-foss-2019b.eb:4]:
```suggestion
homepage = 'https://geant4.cern.ch/'
```

File: easybuild/easyconfigs/g/Geant4/Geant4-10.6-foss-2019b.eb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = ['http://geant4.cern.ch/support/source']
+sources = ['%(namelower)s.%(version_major)s.0%(version_minor)s.tar.gz']
+checksums = ['1424c5a0e37adf577f265984956a77b19701643324e87568c5cb69adc59e3199']
+
+builddependencies = [('CMake', '3.15.3')]
+dependencies = [
+    ('expat', '2.2.7'),
+    # recommended CLHEP version, see http://geant4-data.web.cern.ch/geant4-data/ReleaseNotes/ReleaseNotes4.10.6.html
```
Comment (lexming) [inline easybuild/easyconfigs/g/Geant4/Geant4-10.6-foss-2019b.eb:18]:
```suggestion
    # recommended CLHEP version, see https://geant4-data.web.cern.ch/geant4-data/ReleaseNotes/ReleaseNotes4.10.6.html
```

File: easybuild/easyconfigs/c/CLHEP/CLHEP-2.4.1.3-foss-2019b.eb [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+description = """The CLHEP project is intended to be a set of HEP-specific foundation and
+ utility classes such as random generators, physics vectors, geometry and linear algebra.
+ CLHEP is structured in a set of packages independent of any external package."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://proj-clhep.web.cern.ch/proj-clhep/dist1/']
```
Comment (lexming) [inline easybuild/easyconfigs/c/CLHEP/CLHEP-2.4.1.3-foss-2019b.eb:14]:
```suggestion
source_urls = ['https://proj-clhep.web.cern.ch/proj-clhep/dist1/']
```

**Comment by lexming** [comment]
@boegel since fixing the URLs is a trivial change I'll commit the changes right away

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node381.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/0d47785a652fe57b43b2b426775af907 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node127.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/81002f53f461faf03922d7f40fe57dff for a full test report.

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Pull request merged**